otwcode/otwarchive
# AO3-4194 Associate fields/labels of request invites form

## Issue

https://otwarchive.atlassian.net/browse/AO3-4194

## Purpose

Associates the fields and labels of the Request Invites form so that when the labels are clicked, the proper field receives focus.

## Testing

1. Enable invite requests
2. Log in
3. Go to /user_invite_requests/new
4. Click on either form label.  The associated field should now have the focus.

## Credit

Jen Rogers (she/her)
